app/Services/Adselect.php

```diff
@@ -67,7 +65,7 @@ public function addImpressions(array $events)
 
     public function getBanners(array $requests)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$requests". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)